leaflet-iiif.js

```diff
@@ -44,8 +44,14 @@ L.TileLayer.Iiif = L.TileLayer.extend({
     // Wait for deferred to complete
     $.when(_this._infoDeferred).done(function() {
 
-      // Try to center the map a bit
-      map.setView([-_this.options.tileSize / 2, _this.options.tileSize / 4], 1);
+      // Find best zoom level and center map
+      var initialZoom = _this._getInitialZoom(map.getSize()),
+        imageSize = _this._imageSizes[initialZoom],
+        sw = map.options.crs.pointToLatLng(L.point(0, imageSize.y), initialZoom),
+        ne = map.options.crs.pointToLatLng(L.point(imageSize.x, 0), initialZoom),
+        bounds = L.latLngBounds(sw, ne);
+
+      map.fitBounds(bounds);
```

Comment:
If you add `true` as the second parameter it should fix the issue I was seeing. 

http://leafletjs.com/reference.html#map-fitboundsoptions 

It should reset the map view.